.cirrus.yml

```diff
@@ -0,0 +1,77 @@
+compute_engine_instance:
+  image_project: freebsd-org-cloud-dev
+  image: family/freebsd-13-1
+  platform: freebsd
+  disk: 100 # Gb
```

Comment:
So, is this an image for building HLS binaries for freebsd? Comment needed what it is used for.